share/functions/__fish_print_packages.fish

```diff
@@ -35,6 +35,17 @@ function __fish_print_packages
         return
     end
 
+    # pkg_info on OpenBSD provides versioning info which we want for
+    # installed packages
+    if begin
+            type -q -f pkg_add
+            and test (uname) = "OpenBSD"
+        end
+	# pkg_info can get blocked if there is another instance of pkg_* running.
```

Comment:
Is this why you're not calling `pkg_info` here? That should probably be stated outright.